trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilityType.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Facility, FACILITY_TYPE, MAPPED_FACILITY_TYPE } from '@ukef/dtfs2-common';
+import { mapFacilityType, mapGefFacilityType } from './mapFacilityType';
+
+const { BOND, LOAN, CASH, CONTINGENT } = FACILITY_TYPE;
+
+describe('mapFacilityType', () => {
+  describe(`when the facility type is ${BOND}`, () => {
+    it('should return the facility bondType', () => {
+      const mockBondFacility = {
+        bondType: 'Bid bond',
```

Comment:
Assume we dont have any constants for `Bid bond`?

Reply:
No, the `bondType` is still a bit of a mystery to me, but it does exist. Unsure what other values it could take

Reply:
We do actually have some constants for `bondType` but they're only in E2E. 

@BethThomas141 If you search for `BSS_FACILITY_TYPE =` and `bondType: ` - might help to demystify? `bondType` is a field in the UI that a maker can select in the Portal `(bond-details.njk)`

I will create a ticket to update instances of `bondType` to use constants